challenges/poetry-club-door-password.md

```
@@ -0,0 +1,52 @@
+# Programming Challenge: [Poetry Club Door Password][door-password]
```

Markdownlint

Missing link or image reference definition: "door-password"
Reference links and images should use a label that is defined

(MD052, reference-links-images)